api/tests/Behat/bootstrap/v2/Common/AssertTrait.php

```diff
@@ -43,6 +43,29 @@ public function assertIntEqualsInt(
         );
     }
 
+    public function assertBoolIsTrue(
+        $expected,
+        string $comparisonSubject
+    ) {
+        assert(
+            true == $expected,
```

Comment:
Not sure what level of weirdness PHP goes to when using `==` vs. `===` but I always tend to err on the side of caution and default to `===` to be sure the strictest assertion is used.

Reply:
Ah thanks!